### PR TITLE
use proper path to create host-config symlink

### PR DESCRIPTION
**Pull request by cyrush**

### CODE REVIEW

File: uberenv.py [hunk 1]
```diff
@@ -1065,12 +1065,13 @@ def install(self):
                     if len(hc_glob) > 0:
                         hc_path  = hc_glob[0]
                         hc_fname = os.path.split(hc_path)[1]
-                        if os.path.islink(hc_fname):
-                            os.unlink(hc_fname)
-                        elif os.path.isfile(hc_fname):
-                            sexe("rm -f {0}".format(hc_fname))
-                        print("[symlinking host config file to {0}]".format(pjoin(self.dest_dir,hc_fname)))
-                        os.symlink(hc_path,hc_fname)
```
Comment (cyrush) [inline uberenv.py:1073]: this guy was the issue, it wasn't use the pjoined path that was actually printed

File: uberenv.py [hunk 2]
```diff
@@ -1065,12 +1065,13 @@ def install(self):
...
+                        hc_symlink_path = pjoin(self.dest_dir,hc_fname);
```
Comment (chapman39) [inline uberenv.py:1068]:
```suggestion
                        hc_symlink_path = pjoin(self.dest_dir,hc_fname)
```

File: uberenv.py (hunk 1 above)
Comment (chapman39) [reply uberenv.py:1073]: my mistake. thank you for catching this

**Review by chapman39 — APPROVED** [review]

**Review by white238 — APPROVED** [review]

**Comment by cyrush** [comment]
thanks!

**Pull request merged**